Sources/DeepLearning/Operators.swift

```diff
@@ -16,8 +16,12 @@
 import TensorFlow
 #endif
 
-// `round` Tensor Float or Double values to the to closest integral values with greater magnitude
-// This is a manual definition.
-func round<T : BinaryFloatingPoint>(_ t: Tensor<T>) -> Tensor<T> {
-    return Raw.round(t)
+// Rounds the values of a tensor to the nearest integer, element-wise.
+func round<Scalar: BinaryFloatingPoint>(_ x: Tensor<Scalar>) -> Tensor<Scalar> {
+    return Raw.round(x)
+}
+
+// Return a tensor with the same shape and contents as input.
+func identity<Scalar>(_ x: Tensor<Scalar>) -> Tensor<Scalar> {
```

Comment:
Do you intend to use `identity` as a default activation function (e.g. default argument to `Dense` initializer)?

FYI: shape errors due to differentiating an identity function activation could be improved: https://bugs.swift.org/projects/TF/issues/TF-243

Reply:
I do not intend to do so because 1) default activation as identity is not common and 2) it's better to show the activation argument in code completion as the first candidate.